compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/binary/KSClassDeclarationDescriptorImpl.kt

```diff
@@ -68,10 +69,14 @@ class KSClassDeclarationDescriptorImpl private constructor(val descriptor: Class
         descriptor.unsubstitutedPrimaryConstructor?.let { KSFunctionDeclarationDescriptorImpl.getCached(it) }
     }
 
+    // Workaround for https://github.com/google/ksp/issues/195
+    private val mockSerializableType = ResolverImpl.instance.mockSerializableType
```

Comment:
Since it is already a singleton, maybe just use that singleton directly?

Reply:
I wanted to minimize the per-call overhead here. I think a reference to a private val is faster than `ResolverImpl.instance.mockSerializableType`?